packages/go_router_builder/example/pubspec.yaml

```diff
@@ -8,8 +8,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  # Uses path dependency override
-  go_router: any
+  go_router:
+    path: ../../go_router
```

Comment:
Okay I changed it back: [7924bb2](https://github.com/flutter/packages/pull/2252/commits/7924bb2117ed3f4c6fe26f35aef620d9b7f2ab43). 

@kevmoo feel free to jump in here, but it seems like this should be okay for most people, who probably just want to download and run the example, since `main` matches the latest release.